.github/workflows/build-filesystem-provider.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build filesystem-provider
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+    paths:
+      - migrations/**
```

Comment:
update path trigger to match where devs would be making changes

Reply:
fixed